renderer/tray/tray.js

```diff
@@ -1,46 +1,49 @@
-const joinForm = document.getElementById("enterCall");
+const joinForm = document.getElementById('enterCall');
 
-joinForm.addEventListener("submit", (event) => {
+const entryID = 'entry';
+const inCallID = 'inCall';
```

Comment:
tiny tiny nit: these seem to be both classes and IDs -- most of the other similar things just got reduced to e.g. "on", so any reason why not just "entry"?

Reply:
I used the `ID` suffix here because we already have an `entry` var for the actual element where this is used, so this avoid a name collision: `const entry = document.getElementById(entryID);`

I could rename `entry` to `entryEle` or something, but since `entryID` is also in a wider scope it seemed more relevant to make _that_ one more descriptive and explicit for anyone who reads it. Where we use `on` and `off`, those vars are defined very close to actual usage and not easily read out of context so it seemed less necessary to make those more verbose. At least that was my original reasoning, happy to change!

Reply:
ah I'd missed the `entry` elsewhere, thanks for sharing the rationale :)